### ISSUE TITLE
Smart Airways

### ISSUE THREAD

**Issue by tracernz** [flybywiresim/a32nx#5331]
### Aircraft Version

Development

### Description

When two successive airways are entered on the AIRWAYS lat rev page that do not have a common waypoint, but they do intersect geometrically, the FMS creates a waypoint at the intersection and prefixes the connecting airway with X as it's name.

### References (optional)

Honeywell Step 1A Pilot's Guide

**Comment by Benjozork** [comment]
Should we hold off and only do this in cFPM ?

**Comment by tracernz** [comment]
> Should we hold off and only do this in cFPM ?

Yes, that is my intent.

**Comment by Benjozork** [comment]
Okay.

**Comment by wpine215** [comment]
Will this be revisited with custom FMS 2.0?

**Comment by tracernz** [comment]
Yep 👍

**Comment by flogross89** [comment]
@tracernz @BlueberryKing Just stumbled over this, don't we have this now? 
EDIT: Sorry for the ping, misread. Seems we can only find existing intersection fixes.

**Comment by tracernz** [comment]
@flogross89 Nope, not currently possible sadly, as we can't fetch arbitrary airways to do geometric intersections.
